File: Serene/Serene.Script/Meeting/MeetingAttendee/MeetingAttendeeGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingAttendee"), IdProperty("MeetingAttendeeId"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingAttendeeDialog)), LocalTextPrefix("Meeting.MeetingAttendee"), Service("Meeting/MeetingAttendee")]
    public class MeetingAttendeeGrid : EntityGrid<MeetingAttendeeRow>
    {
        public MeetingAttendeeGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingAttendeeRow
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingAttendee/MeetingAttendeeGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingAttendeeGrid extends Serenity.EntityGrid<MeetingAttendeeRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingAttendee'; }
        protected getDialogType() { return MeetingAttendeeDialog; }
        protected getIdProperty() { return MeetingAttendeeRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAttendeeRow.localTextPrefix; }
        protected getService() { return MeetingAttendeeService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}